lib/chiketto/event.rb

```diff
@@ -16,11 +16,8 @@ class Event < Resource
               :changed
 
     def self.create(params)
-      params[:title] = params.delete(:name) if params[:name]
-      response = Event.post 'event_new', params
-      if response.fetch('process', false)
-        Event.find response['process']['id']
-      end
+      response = Event.post 'events', params
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.